modules/oe_translation_cdt/modules/oe_translation_cdt_mock/src/Plugin/ServiceMock/FileApi.php

```diff
@@ -73,10 +73,10 @@ public static function create(ContainerInterface $container, array $configuratio
   /**
    * {@inheritdoc}
    */
-  protected function getEndpointUrl(): string {
+  protected function getEndpointUrlPath(): string {
     // The file URL is not present in the static settings.
     // It is defined by the Status API response.
-    return 'https://example.com/api/files/:language/:id';
+    return '/v2/files/:language/:id';
```

Comment:
This is the only place where this address is hardcoded, as it is variable and it depends on the mock response.